Update android-configure.md (#12413)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/android-configure.md

Using this privacy control won't impact the device compliance check or conditional access. For example, devices with a malicious app will always have a risk level of "Medium".

## Disable sign-out

> Defender for Endpoint supports deployment without sign-out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering the device.
> Use the following steps to configure Disable sign-out:

Reviewer: ```suggestion
Defender for Endpoint supports deployment without the sign-out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering with the device.
Use the following steps to configure Disable sign-out:
```

1. In [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Apps** > **App configuration policies** > **Add** > **Managed devices**.

> 2. Give the policy a name,select **Platform > Android Enterprise**, and select the profile type.

Reviewer: ```suggestion
2. Give the policy a name, select **Platform > Android Enterprise**, and select the profile type.
```

Using this privacy control won't impact the device compliance check or conditional access. For example, devices with a malicious app will always have a risk level of "Medium".

> ## Disable sign out
> Defender for Endpoint supports deployment without sign out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering the device.

Reviewer: ```suggestion
## Disable sign-out

Defender for Endpoint supports deployment without the sign-out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering with the device.
```

Author: done

> Use the following steps to configure the Disable sign out:

Reviewer: ```suggestion
Use the following steps to configure Disable sign-out:
```

Author: done

> 1. In [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431) and go to **Apps** > **App configuration policies** > **Add** > **Managed devices**.
> 2. Give the policy a name, **Platform > Android Enterprise**, select the profile type.

Reviewer: ```suggestion
1. In [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Apps** > **App configuration policies** > **Add** > **Managed devices**.
2. Give the policy a name, select **Platform > Android Enterprise**, and select the profile type.
```

Author: done

3. Select **Microsoft Defender for Endpoint** as the target app.

> 4. In Settings page, select **Use configuration designer** and add **Disable Sign Out** as the key and value type as **Integer**
>    - By default, Disable Sign Out = 1 for Android enterprise personally owned work profile, fully managed, company owned personally enabled profiles and 0 for device administrator mode.
>    - Admin needs to make Disable Sign Out = 0 to enable the sign-out button in the app. Users will be able to see the sign out button once the policy is pushed.
> 5. Select **Next** and assign this profile to targeted devices/users.

Reviewer: ```suggestion
4. In the Settings page, select **Use configuration designer** and add **Disable Sign Out** as the key and **Integer** as the value type.
   - By default, Disable Sign Out = 1 for Android Enterprise personally owned work profiles and fully managed, company-owned personally enabled profiles and 0 for device administrator mode.
   - Admins need to make Disable Sign Out = 0 to enable the sign-out button in the app. Users will be able to see the sign-out button once the policy is pushed.
5. Select **Next** and assign this profile to targeted devices and users.
```

Author: done
